docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -33,17 +33,14 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
   - **ORDER BY**
   - **LIKE**
 
-- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
-  Use wildcards with the **LIKE** operator to query possible values:
-
-  ```bash
-  // Create a "ProductId" custom Search Attribute of type String
-  tctl admin cluster add-search-attributes --name ProductId --type String
-  // Match values that start with "book"
-  ProductId LIKE "book%"
-  // Match values that contain "favorite"
-  ProductId LIKE "%favorite%"
+- To search for a specific token, set a custom Search Attribute of type `Text` that has the value of the token.
+  To search for multiple tokens, separate them by spaces or punctuation.
+  In the following example, the Query runs after setting the custom Search Attribute to `type1`:
```

Comment:
We should link to examples in each lang + the CLI tool

Reply:
I'll link to the examples and tctl. I'll be sure to change it to CLI when that's all finally settled

Reply:
Links added. One for tctl-next and one to the Observability section that has code examples in tabs